aptos-move/framework/aptos-framework/sources/staking_contract.move

```diff
@@ -722,6 +727,23 @@ module aptos_framework::staking_contract {
         pool_u64::update_total_coins(distribution_pool, updated_total_coins);
     }
 
+    /// Create the seed to derive the resource account address.
+    fun create_resource_account_seed(
```

Comment:
create indicates you are "creating" something indicating something non pure. Perhaps compute_resource_account_seed